backend/typescript/models/baseCamp.model.ts

```diff
@@ -0,0 +1,59 @@
+import { Schema, Document, model } from "mongoose";
+import { FormQuestion } from "./formQuestion.model";
+
+export interface BaseCamp extends Document {
+  id: string;
+  ageLower: number;
+  ageUpper: number;
+  name: string;
+  description: string;
+  location: string;
+  fee: number;
+  formQuestions: FormQuestion[] | Schema.Types.ObjectId[];
+  camps: Schema.Types.ObjectId[];
+}
+
+const BaseCampSchema: Schema = new Schema({
```

Comment:
Oops, was there a merge conflict issue? I believe we renamed this to just "Camp"

Reply:
ya, just realised it's a merge conflict issue. Will look into this.